operator/seldon-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: seldon
+    app.kubernetes.io/instance: seldon1
+    app.kubernetes.io/name: seldon
+    app.kubernetes.io/version: v0.5
```

Comment:
Is this supposed to be the kubernetes version?

Reply:
good catch this is from the kustomize. Will change there.